pages/textbook-beta/course/quantum-machine-learning-course.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <main class="quantum-machine-learning-course-page">
+    <TextbookBetaContentMenuSection
+      class="quantum-machine-learning-course-page__content-menu"
+    />
+    <AppPageHeaderWithImage :cta="startLearningCTA" :back-link="backToTextbookHomeLink">
+      <template slot="title">
+        {{ headerTitle }}
+      </template>
+      <template slot="description">
+        <p
+          v-for="(paragraph, index) in headerDescription"
+          :key="index"
+          v-text="paragraph"
+        />
+      </template>
+      <template slot="image">
+        <img class="app-page-header-with-img__image" :src="headerImg">
+      </template>
+    </AppPageHeaderWithImage>
+    <PrerequisiteMaterialSection class="quantum-machine-learning-course-page__section" :data="quantumMachineLearningCoursePrerequisites" />
+    <CoursePagesSection class="quantum-machine-learning-course-page__section" :courses="courses" :img-base="imageUrlBase" />
+  </main>
+</template>
+
+<script lang="ts">
+import { Course, Prerequisite } from 'constants/textbookBetaContent'
+import { Component } from 'vue-property-decorator'
+import QiskitPage from '~/components/logic/QiskitPage.vue'
+import { GeneralLink, textbookBetaQuantumMachineLearningCourseStartLearningUrl } from '~/constants/appLinks'
+
+@Component({
+  layout: 'default-max',
+  head () {
+    return {
+      title: 'Quantum machine learning course'
+    }
+  }
+})
+export default class QuantumMachineLearningCoursePage extends QiskitPage {
+  routeName: string = 'quantum-machine-learning-course'
+
+  headerTitle = 'Quantum machine learning'
+  headerDescription = [`This course contains around eight hours of content, and
```

Comment:
since the `headerDescription` is only one paragraph, we can remove the array brackets `[` and `]`, leaving us with:

```js
headerDescription = `This course contains around eight hours of content, and
  is aimed at self-learners who are comfortable with undergraduate-level
  mathematics and quantum computing fundamentals. This course will take you
  through key concepts in quantum machine learning, such as parameterized
  quantum circuits, training these circuits, and applying them to basic
  problems. By the end of the course, you'll understand the state of the field,
  and you'll be familiar with recent developments in both supervised and
  unsupervised learning such as quantum kernels and general adversarial networks.
  This course finishes with a project that you can use to showcase what you've
  learnt.`
```